test/integration/space-members-integration.js

```diff
@@ -0,0 +1,19 @@
+export default function spaceMembersTests (t, space) {
+  t.test('Gets spaceMembers', t => {
+    t.plan(3)
+    return space.getSpaceMembers().then(response => {
+      t.ok(response.sys, 'sys')
+      t.ok(response.sys.type, 'Array')
+      t.ok(response.items, 'items')
```

Comment:
this can be true even if it returns nothing, right? should we check that it returns something or is that overkill

Reply:
hmm good point, I can test that returns the correct type for one of the items? could also test the length, will add